woocommerce/woocommerce-android
# Enable Products M4 for all

- Removed `PRODUCT_RELEASE_M4 ` feature flag
- Hid beta features row in settings 
- Updated the product banner to always show the products tab top banner copy (title & info) for M4 features
- Removed conditional check for Add Product FAB visibility

## Testing

- On `develop` or any other branch, first launch the app and turn off the Products feature switch in Settings > Betal Features --> there should be no Add product FAB in the Products tab and the top banner does not mention product creation
- On PR branch, launch the app
- Go to the Products tab --> the Add product FAB should be visible even though the feature switch was turned off previously, and the top banner should mention product creation
- Go to Settings --> Beta Features row should not be visible

Update release notes:
